nari/ext/act/parser/ability.py

```diff
@@ -20,7 +61,7 @@ def action_effect_from_logline(params: list[str]) -> ActionEffect:
     param0, param1, severity, effect_category, value, flags, multiplier = parsed_params
     return ActionEffect(effect_category=effect_category, severity=severity, flags=flags, value=value, multiplier=multiplier, additional_params=[param0, param1])
 
-def ability_from_logline(timestamp: Timestamp, params: list[str]) -> Event:
+def ability_from_logline(timestamp: Timestamp, params: list[str]) -> Ability:
```

Comment:
Ultra nit: does this even work? AFAIK the parser will take any implementation of Event but I'm not sure if it needs to actually be typed as Event. Seems fine either way, even if a bit of a waste of a function.